### PR TITLE
unbreak the stuff that @Happy-dude broke

### PR DESCRIPTION
**Pull request by bfredl**
the things that you need to deal with..

### CODE REVIEW

File: nvim/lua/bfredl/colors.lua [hunk 1]
```diff
@@ -65,7 +65,7 @@ function h.fastmode() -- {{{
 
   b.exec [[
   augroup fastmode
-    au InsertLeave,TextChanged <buffer>  * lua _G._ccheck()
```
Comment (bfredl) [inline nvim/lua/bfredl/colors.lua:68]: not a good look sir

Comment (happy-dude) [reply nvim/lua/bfredl/colors.lua:68]: What does `_G._ccheck()` do?

Comment (bfredl) [reply nvim/lua/bfredl/colors.lua:68]: who will ever know for sure?

Comment (happy-dude) [reply nvim/lua/bfredl/colors.lua:68]: 🧐 good enough for me.

**Review by happy-dude — APPROVED** [review]
I approve these changes.

Unsure how it's related to my change, but smiles have been known to break terminals.

Note: it would be cool for GitHub to have icons for ✅
Like powerline stuff.

**Pull request merged**